modules/improvedigitalBidAdapter.js

```diff
@@ -101,6 +101,7 @@ export const spec = {
    * @return {Bid[]} An array of bids which were nested inside the server.
    */
   interpretResponse: function (serverResponse, {bidderRequest}) {
+    const razr = new RAZR();
```

Comment:
I don't think it's a good idea to create a new instance on each interpretResponse. There can be many slots on page refreshing (loading new ads) many times during the same session (for example on a game page while user is playing a game). The effect of the current implementation is not only higher memory consumption but also for example loading renderer.js multiple times when there is more than one RAZR add in the same session but different refresh (i.e. separate interpretResponse calls.